src/test/java/com/artipie/docker/DockerOnPortIT.java

```diff
@@ -4,125 +4,126 @@
  */
 package com.artipie.docker;
 
-import com.artipie.ArtipieServer;
 import com.artipie.RepoConfigYaml;
-import com.artipie.docker.junit.DockerClient;
-import com.artipie.docker.junit.DockerClientSupport;
-import java.io.IOException;
-import java.net.ServerSocket;
+import com.artipie.test.ContainerResultMatcher;
+import com.artipie.test.TestDeployment;
 import java.nio.file.Path;
-import java.util.Arrays;
-import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.AllOf;
+import org.hamcrest.core.IsEqual;
 import org.hamcrest.core.StringContains;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 import org.junit.jupiter.api.io.TempDir;
 
 /**
  * Integration test for local Docker repository running on port.
  *
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @since 0.10
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-@DockerClientSupport
-@Disabled("FIXME: migrate artipie to testcontainers")
 final class DockerOnPortIT {
 
     /**
-     * Example docker image to use in tests.
+     * Temp directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
      */
-    private Image image;
+    @TempDir
+    static Path temp;
 
     /**
-     * Docker client.
+     * Repository port.
      */
-    private DockerClient client;
+    private static final int PORT = 8085;
```

Comment:
Is it internal port (used in internal Docker network), if not it could be not available on test machine or used by another process.

Reply:
Yes, this is the port inside a docker container.